src/platform_impl/apple/uikit/view.rs

```diff
@@ -454,25 +457,18 @@ impl WinitView {
         for touch in touches {
             let logical_location = touch.locationInView(None);
             let touch_type = touch.r#type();
-            let force = if os_supports_force {
+            let force = if let UITouchType::Pencil = touch_type {
+                None
```

Comment:
Hmm, I think this is wrong? `UITouchType::Pencil` is exactly the kind of device that _does_ support force - even if we end up not using it below, I'd still rather compute the force, because we will need it later when we add proper support for pencils (and then this kind of branch is very confusing to see).

Reply:
Same as https://github.com/rust-windowing/winit/pull/3876#discussion_r1721053489.

I can't use force here because I can't encode it into `PointerType/Source`, the `Unknown` variant can't hold `Force` and we can't just report it as `Touch`.

As mentioned above I'm happy to introduce `PointerType/Source::Pen` here just for MacOS and Windows.

Reply:
Yeah, I got that now, thanks!